src/styles/components/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  display: grid;
+  padding: 50px 40px;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+@media (max-width: 1023px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), media queries should be ordered from smaller to larger. The media query for 'max-width: 1023px' should be placed after the ones for 'max-width: 677px' and 'max-width: 487px'.